packages/devtools-launchpad/.babelrc

```diff
@@ -10,7 +10,7 @@
 
     ["module-resolver", {
       "alias": {
-        "react-dom": "react-dom/dist/react-dom",
+        "react-dom": "react-dom/umd/react-dom.development",
```

Comment:
So the issue is due to a custom module system (per #23)? I'm not sure what “single module plugin” is.

I guess if it works around a bug for you, that’s cool, I just wanted to verify. Please not `react.development` is *not* suitable for production.